### ISSUE TITLE
Build fails on macOS

### ISSUE THREAD

**Issue by ffflorian** [neon-bindings/neon#188]
I created a new project with `neon new helloworld` which was successfully created, but after running `npm install`, I get the error
```
error: failed to run custom build command for `neon-runtime v0.1.12`
process didn't exit successfully: `/Users/florian/git/helloworld/native/target/release/build/neon-runtime-dffb6e00c22f746e/build-script-build` (exit code: 101)
```

__My environment__:
`neon-cli` 0.1.12
`rustc` 1.15.1
`npm` 4.1.2
Node.js 7.5.0
Xcode 8.2.1
macOS 10.12.3

Note: It works on Ubuntu 16.10 with the same `rustc` and `neon-cli` versions.

----------------------
Extended log:
```
[...]
8010 silly build helloworld@0.1.0
8011 info linkStuff helloworld@0.1.0
8012 silly linkStuff helloworld@0.1.0 has /Users/florian/git as its parent node_modules
8013 verbose linkBins helloworld@0.1.0
8014 verbose linkMans helloworld@0.1.0
8015 silly install helloworld@0.1.0
8016 info lifecycle helloworld@0.1.0~install: helloworld@0.1.0
8017 verbose lifecycle helloworld@0.1.0~install: unsafe-perm in lifecycle true
8018 verbose lifecycle helloworld@0.1.0~install: PATH: /Users/florian/.npm-packages/lib/node_modules/npm/bin/node-gyp-bin:/Users/florian/git/helloworld/node_modules/.bin:/Users/florian/.cargo/bin:/Users/florian/.yarn/bin:/Users/florian/.npm-packages/bin:/usr/local/bin:/Users/florian/bin:/Users/florian/bin/emscripten:/Users/florian/.yarn/bin:/Users/florian/.npm-packages/bin:/usr/local/bin:/Users/florian/bin:/Users/florian/bin/emscripten:/Users/florian/.yarn/bin:/Users/florian/.npm-packages/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin
8019 verbose lifecycle helloworld@0.1.0~install: CWD: /Users/florian/git/helloworld
8020 silly lifecycle helloworld@0.1.0~install: Args: [ '-c', 'neon build' ]
8021 silly lifecycle helloworld@0.1.0~install: Returned: code: 1  signal: null
8022 info lifecycle helloworld@0.1.0~install: Failed to exec install script
8023 warn helloworld@0.1.0 No description
8024 verbose If you need help, you may report this error at:
8024 verbose     <https://github.com/npm/npm/issues>
8025 warn helloworld@0.1.0 No repository field.
8026 verbose If you need help, you may report this error at:
8026 verbose     <https://github.com/npm/npm/issues>
8027 verbose stack Error: helloworld@0.1.0 install: `neon build`
8027 verbose stack Exit status 1
8027 verbose stack     at EventEmitter.<anonymous> (/Users/florian/.npm-packages/lib/node_modules/npm/lib/utils/lifecycle.js:279:16)
8027 verbose stack     at emitTwo (events.js:106:13)
8027 verbose stack     at EventEmitter.emit (events.js:192:7)
8027 verbose stack     at ChildProcess.<anonymous> (/Users/florian/.npm-packages/lib/node_modules/npm/lib/utils/spawn.js:40:14)
8027 verbose stack     at emitTwo (events.js:106:13)
8027 verbose stack     at ChildProcess.emit (events.js:192:7)
8027 verbose stack     at maybeClose (internal/child_process.js:890:16)
8027 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
8028 verbose pkgid helloworld@0.1.0
8029 verbose cwd /Users/florian/git/helloworld
8030 error Darwin 16.4.0
8031 error argv "/usr/local/Cellar/node/7.5.0/bin/node" "/Users/florian/.npm-packages/bin/npm" "install"
8032 error node v7.5.0
8033 error npm  v4.1.2
8034 error code ELIFECYCLE
8035 error helloworld@0.1.0 install: `neon build`
8035 error Exit status 1
[...]
```

**Comment by dherman** [comment]
I ran into a macOS issue yesterday halfway through publishing all the crates, so things were out of sync for a while until late last night. I can't quite tell from reading your logs but it's possible that's what's wrong. Could you just try again one more time and see if by chance the problem fixed itself?

But if not, we'll dig in further (feel free to join the Slack if you want me to help you troubleshoot)!

**Comment by dherman** [comment]
Fixed by #189 (and released in neon 0.1.13).
